lib/scarlet/bot.rb

```diff
@@ -29,6 +43,10 @@ def initialize
     use Scarlet::Plugins::Command
   end
 
+  def signals
+    Signals
```

Comment:
Get rid of this indirection. You're just hiding the class, it's still a singleton. Same goes for `Scarlet::Logger`, we should really get rid of that.